recipes-bsp/xt-rpi-u-boot-scr/files/boot.cmd.xen.1.in

```diff
@@ -1,6 +1,4 @@
 fdt addr ${fdt_addr}
```

Comment:
Comment regarding the misprint in the commit message. Seems it should be `to generate` not `go generate`.

Reply:
fixed